core/src/main/java/org/ehcache/config/writebehind/DefaultWriteBehindConfiguration.java

```diff
@@ -25,7 +27,7 @@
  */
 public class DefaultWriteBehindConfiguration implements WriteBehindConfiguration {
 
-  private int minWriteDelay = 1;
+  private int minWriteDelay = 0;
   private int maxWriteDelay = 1;
```

Comment:
I'm not sure it makes sense that these values have defaults... getting déjà vu here.